examples/kubernetes/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: Deploy Optimum for Intel® Gaudi® Accelerators Examples to Kubernetes
```

Comment:
@dmsuehir, please follow the best practices for the name, here you added description instead of the name. Also the directory containing the chart must have the same name as the chart itself